xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -5229,12 +5229,13 @@ glusterd_import_friend_volumes_synctask(void *opaque)
 {
     int32_t ret = -1;
     int32_t count = 0;
-    int i = 1;
+    int i = 0; /* Always start from 0 to access correct bitmap */
```

Comment:
I wouldn't change this. If you do that, a lot of other places that consider that '1' is the first volume will need to be adjusted.